### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kapesha**
- [DEMO LINK](https://kapesha.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://kapesha.github.io/layout_product-cards/report/html_report/)

X there 2 BEM blocks card and stars each in it own file
X SCSS Nesting is used for elements, modifiers and pseudo-classes
X SCSS Variables are used for main values and placed in a separate file
X all stars--N modifiers work as expected (Highlight first N stars)
X Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Test report link is not valid, try to redeploy it.

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,58 @@
...
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__num-of-reviews">Reviews: 5</div>
+          </div>
+        </div>
+        <div class="card__buy">
+          <div class="card__price">
+            <div class="card__text-price">Price</div>
+            <div class="card__amount">$2,199</div>
```
Comment (loralevitska) [inline src/index.html:46]: Use `p` tag for this text

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,58 @@
...
+          </div>
+          <a
+            href="#"
+            class="card__buy-button"
+            data-qa="hover"
+            >buy
+          </a>
```
Comment (loralevitska) [inline src/index.html:53]:
```suggestion
            data-qa="hover"
          >
            buy
          </a>
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,58 @@
...
+          </div>
+          <div class="card__reviews">
+            <div class="card__stars stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__num-of-reviews">Reviews: 5</div>
```
Comment (loralevitska) [inline src/index.html:40]: Use `p` according semantic rules

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+.card {
+  @include flex-column;
+
+  box-sizing: border-box;
+  background-color: white;
+  width: 200px;
+  height: 408px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #F3F3F3;
```
Comment (loralevitska) [inline src/styles/blocks/card.scss:19]: It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+.card {
+  @include flex-column;
+
+  box-sizing: border-box;
+  background-color: white;
+  width: 200px;
+  height: 408px;
+  padding-top: 32px;
+  padding-bottom: 16px;
```
Comment (loralevitska) [inline src/styles/blocks/card.scss:18]: Combine with `padding` shorthand property. Fix all cases

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+  }
+
+  &__stars {
+    display: flex;
+  }
+
+  &__num-of-reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```
Comment (loralevitska) [inline src/styles/blocks/card.scss:76]: Remove redundant default value, fix all cases
```suggestion
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
You have wrong links. Try to redeploy your solution 
Here should be your task solution, not task requirement 
image
and report link also doesn't work
image

If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Pull request closed without merging**